README.md

```diff
@@ -1,10 +1,55 @@
-# default-TEM-model
+# model_seg_mouse_axon-myelin_tem
+---
+## Model overview
+![tem model preview image](tem_model_preview.png)
+
 AxonDeepSeg default TEM model and testing image. This model is suited for a resolution of 0.01 micrometer per pixel and was trained on mouse brain data collected via a Transmission Electron Microscope (TEM).
 
+## Dependencies
+- [ivadomed](https://ivadomed.org/) commit: e6554281d07b5afef9e68ce8b02e86b02bd68363
+- [axondeepseg](https://axondeepseg.readthedocs.io/en/latest/) commit: 805868e39eddf620c9f3d60d313cadffb1121bfb
+
+## Segment (ADS)
+To segment an image using this model, use
+```
+axondeepseg -t TEM -i <IMG_PATH> -m <path_to_model_folder> -s <PIXEL_SIZE>
```

Comment:
In the SEM readme, we removed the `-m` option from the CLI line.
I'm not sure which one makes more sense but we should make the readme files consistent.

Reply:
Ah yes forgot to remove it thanks for the catch